google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/QueryParameterValue.java

```diff
@@ -245,20 +248,43 @@ public static QueryParameterValue date(String value) {
   }
 
   /**
+   * Creates a {@code QueryParameterValue} object with a type of DATE.
+   */
+  public static QueryParameterValue date(CivilDate value) {
+    return of(value, StandardSQLTypeName.DATE);
+  }
+
+  /**
    * Creates a {@code QueryParameterValue} object with a type of TIME. Must be in the format
    * "HH:mm:ss.SSSSSS", e.g. "12:41:35.220000".
    */
   public static QueryParameterValue time(String value) {
```

Comment:
I think I do. Sometimes time in string format might be passed through from another source, and it can be really convenient for testing purposes (it looks a lot simpler).